backend/docker/sql/backend-dev.sql

```diff
@@ -384,3 +385,31 @@ CREATE TRIGGER transfer_count_trunc AFTER TRUNCATE ON extrinsic
 -- initialize the counter table
 UPDATE total SET count = (SELECT count(*) FROM extrinsic WHERE method IN ('transfer', 'transferKeepAlive')) WHERE name = 'transfers';
 COMMIT;
+
```

Comment:
Remove it from backend.sql for now. We have it in migrations.

Reply:
Removed